### PR TITLE
[Doc] Tutorial on exporting TorchRL models

### PR DESCRIPTION
**Pull request by vmoens**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #2559
* __->__ #2557

cc @agunapal @matteobettini @desertfire @angelayi @gmagogsfm @ezyang (and @janeyx99 since it's kinda BE? 😉 )

### CODE REVIEW

File: tutorials/sphinx-tutorials/export.py [hunk 1]
```diff
@@ -0,0 +1,393 @@
...
+# Here's an example of how you can use AOTInductor to export your policy, inspired by the
+# `AOTI documentation <https://pytorch.org/docs/main/torch.compiler_aot_inductor.html>`_:
+#
+
+from torch._inductor.package import load_package
+
+with TemporaryDirectory() as tmpdir:
+    path = str(Path(tmpdir) / "model.pt2")
+    with torch.no_grad():
+        so_path = torch._inductor.aoti_compile_and_package(
```
Comment (angelayi) [inline tutorials/sphinx-tutorials/export.py:254]: This should just give you the package path, not the so path.

File: tutorials/sphinx-tutorials/export.py [hunk 2]
```diff
@@ -0,0 +1,393 @@
...
+            exported_policy,
+            args=(),
+            kwargs={"pixels": pixels},
+            # Specify the generated shared library path
+            package_path=path,
+        )
+
+    compiled_module = load_package(str(Path(tmpdir) / "model.pt2"))
```
Comment (angelayi) [inline tutorials/sphinx-tutorials/export.py:262]: You can also use `torch._inductor.aoti_load_package` which is a wrapper around `load_package`

**Review by angelayi — APPROVED** [review]
Thanks for writing this up!!

File: tutorials/sphinx-tutorials/export.py (hunk 1 above)
Comment (vmoens) [reply tutorials/sphinx-tutorials/export.py:265]: yeah I mixed up two docs (stable and main) 
glad you caught it!

**Comment by vmoens** [comment]
> Thanks for writing this up!!

Thanks for the feature! Extra useful

File: tutorials/sphinx-tutorials/export.py [hunk 3]
```diff
@@ -0,0 +1,402 @@
...
+# function that will read our input tensor and output an action without any reference to TorchRL or tensordict modules.
+#
+
+fake_td = env.base_env.fake_tensordict()
+pixels = fake_td["pixels"]
+with set_exploration_type("DETERMINISTIC"):
+    exported_policy = torch.export.export(
+        policy_transform.select_out_keys("action"),
+        args=(),
+        kwargs={"pixels": pixels},
```
[the remark this answers is not in the document]
Comment (vmoens) [reply tutorials/sphinx-tutorials/export.py:185]: hmm yeah we could show that it works with stack of pixels! I'll make a PR with that

**Pull request merged**